pkg/dagger.io/dagger/core/fs.cue

```diff
@@ -60,7 +60,7 @@ import "dagger.io/dagger"
 	// Contents to write
 	contents: string
 	// Permissions of the file
-	permissions: *0o600 | int
+	permissions: *0o644 | int
```

Comment:
This is setting the permission for the created file, right?
Shouldn't it technically be `0o755` then so that it is executable too?

Does seem to work with `0o644` though so maybe more of a nitpick (or maybe incorrect suggestion altogether)

Reply:
`bash.#Run` is actually calling `bash <script file.sh>` (rather than `./<script file.sh>`), so the file doesn't need to be executable per se